swarm.cabal

```diff
@@ -154,6 +164,8 @@ library
                       unordered-containers          >= 0.2.14 && < 0.3,
                       vector                        >= 0.12 && < 0.13,
                       vty                           >= 5.33 && < 5.36,
+                      warp                          >= 3.2,
+                      wai                           >= 3.2,
```

Comment:
We might want to set some upper bounds so that the build does not unexpectedly fail when some breaking changes are released in the packages. :slightly_smiling_face: